src/Tests/Microsoft.NET.ToolPack.Tests/GivenThatWeWantToDefaultPackAToolProjectWithPackagedShim.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.NET.TestFramework;
+using Microsoft.NET.TestFramework.Assertions;
+using FluentAssertions;
+using Xunit;
+using Xunit.Abstractions;
+using NuGet.Packaging;
+using NuGet.Frameworks;
+using fixture = Microsoft.NET.ToolPack.Tests.DefaultPackWithShimsAndResultNugetPackageNuGetPackagexFixture;
+
+namespace Microsoft.NET.ToolPack.Tests
+{
+    public class GivenThatWeWantToDefaultPackAToolProjectWithPackagedShim : SdkTest, IClassFixture<fixture>
```

Comment:
Nit: I would use a PascalCased alias for the fixture (and probably make it slightly more descriptive)

Reply:
I'm not sure what the difference between the new class name (`GivenThatWeWantToDefaultPackAToolProjectWithPackagedShim`) and the existing one (`GivenThatWeWantToPackAToolProjectWithPackagedShim`).  What does it mean to "default pack" a tool project?

I would suggest just leaving all these tests in the same class.  Some of the tests won't use the fixture, but that's fine.

Reply:
Change the fixture name to NupkgOfPackWithShimsFixture and not use alias. And I merged it into GivenThatWeWantToPackAToolProjectWithPackagedShim